ValidationLibrary.Rules/FixableRuleBase.cs

```diff
@@ -14,10 +15,12 @@ namespace ValidationLibrary.Rules
     /// </summary>
     public abstract class FixableRuleBase<T> : IValidationRule where T : IValidationRule
     {
+        [JsonProperty("PullRequestTitle")]
```

Comment:
Don't put `JsonProperties` here, this is not a DTO. Implement separate DTO in `ValidationLibrary.AzureFunctions`

Reply:
Removed